pkg/ioutils/readers.go

```diff
@@ -50,11 +52,11 @@ func NewReaderErrWrapper(r io.Reader, closer func()) io.Reader {
 	}
 }
 
-// bufReader allows the underlying reader to continue to produce
+// BufReader allows the underlying reader to continue to produce
 // output by pre-emptively reading from the wrapped reader.
-// This is achieved by buffering this data in bufReader's
+// This is achieved by buffering this data in BufReader's
 // expanding buffer.
-type bufReader struct {
+type BufReader struct {
```

Comment:
@cpuguy83  Because `NewBufReader` and `NewBufReaderWithDrainbufAndBuffer` will return a `bufReader`, and `golint` will show warnings if `bufReader` is not exported